### PR TITLE
update convert_sync_batchnorm docs

### PR DESCRIPTION
**Pull request by ssnl**
fix some inaccuracies

### CODE REVIEW

File: torch/nn/modules/batchnorm.py [hunk 1]
```diff
@@ -467,28 +467,31 @@ def forward(self, input):
...
-        r"""Helper function to convert `torch.nn.BatchNormND` layer in the model to
-        `torch.nn.SyncBatchNorm` layer.
+        r"""Helper function to convert all :attr:`BatchNorm*D` layers in the model to
+        :class:`torch.nn.SyncBatchNorm` layers.
 
         Args:
-            module (nn.Module): containing module
+            module (nn.Module): module containing one or more attr:`BatchNorm*D` layers
             process_group (optional): process group to scope synchronization,
         default is the whole world
```
Comment (mrshenli) [inline torch/nn/modules/batchnorm.py:476]: I will add a followup to fix this indention

Screen Shot 2020-06-08 at 10 31 50 AM

**Review by mrshenli — APPROVED** [review]
Thanks for fixing!

File: torch/nn/modules/batchnorm.py (hunk 1 above)
Comment (SsnL) [reply torch/nn/modules/batchnorm.py]: fixed!

Comment (mrshenli) [reply torch/nn/modules/batchnorm.py]: Thanks!
